app/controllers/posts_controller.rb

```diff
@@ -14,17 +14,24 @@ def create
     if @post.save
```

Comment:
- [ ] In your posts controller, we need to use CanCanCan for Post authorization. Kindly add this in your Post controller at the top just after the class definition ```load_and_authorize_resource```.